src/pages/saksbehandling/stans/stans.module.less

```diff
@@ -0,0 +1,59 @@
+@import '~/styles/variables.less';
+
+.tittel {
+    width: 100%;
+    padding: @spacing-xs 10rem;
+    border-bottom: 1px solid @navBorder;
+}
+
+.pageContainer {
+    display: flex;
+    flex-direction: column;
+    width: 100%;
+
+    .content {
+        padding: @spacing-xs 10rem;
+    }
+
+    .select {
+        max-width: 16rem;
+    }
+
+    .datepicker {
+        margin-top: @spacing-s;
+    }
+
+    .begrunnelse {
+        margin-top: @spacing-s;
+        max-width: 24rem;
+    }
```

Comment:
Her var det også mye tilsynelatende tilfeldige bredder og paddings. Siden det også er identisk med den andre siden så burde du se på om det kan gjøres felles på en måte.
Hvis skjemaene er identiske; kunne du kanskje dratt ut det til en komponent?

Reply:
Jepp, lagde en felles oppsummeringskomponent for de to fordi de er så like. Men de har ikke samme input så där blir det lite forskjell. Men kan se om jag kan endre og bruke noe standard spacing.